host/config/chain_spec_list_default.json

```diff
@@ -56,7 +56,7 @@
         "l1_contract": null,
         "l2_contract": null,
         "rpc": "https://ethereum-holesky-rpc.publicnode.com",
-        "beacon_rpc": "https://api.holesky.blobscan.com",
+        "beacon_rpc": "https://fabled-weathered-cherry.ethereum-holesky.quiknode.pro/8f1c66935fa5f9afbda0db43318fe3c9e7b061e1/",
```

Comment:
Should we hide this one into maybe CI secrets to prevent unauthorized accesses from others?

Reply:
Maybe we should have a dedicated account for our CI. I'm not paying for this one yet because I just picked the free tier but can buy more capacity if needed. I guess we can see if it's a problem or not, hopefully there's enough free rpcs out there that people won't abuse this one.